src/lazyfunc/__init__.py

```diff
@@ -1 +1,7 @@
+from importlib.metadata import version
+
+__version__ = version("lazyfunc")
+
+# __all__ = ["__version__"]
```

Comment:
Something like this:

```suggestion
__version__ = version("lazyfunc")

__all__ = ["__version__", "LazyFunc"]
```

Reply:
Nice, I shall commit this one and resolve